app/services/preview_secret_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# For more info, see https://docs.publishing.service.gov.uk/manual/content-preview.html#authentication
+class PreviewSecretService
```

Comment:
I feel sad suggesting this, as it's rather fun that we have a class called secret service, but since this doesn't create something that is actually secret it might be clearer to call it PreviewAuthBypassService?

Reply:
I'm a little confused about this, so would be good to clarify here. 

My impression of the draft stack is that it is protected by some kind of auth (secret), whether that's via Signon, or via auth bypass tokens. This makes sense especially for access limiting, where you could reasonably call the content/attachment 'secret' until it's published; in other cases, perhaps it's not 'secret', but it is 'confidential' among publishers. 

I can see the logic with 'AuthBypassService', but perhaps a downside with this name is that it suggests some external operation is being performed, but in reality it's more of a generator. Although I'm not too fussed about the name, is 'secret' good enough?

Reply:
Probably worth having a quick chat about this, will try grab you shortly.

Reply:
OK. I've gone with renaming to 'AuthBypass' - I couldn't decide between them, so hopefully this makes more sense :-).